vhdbuilder/scripts/windows/collect-windows-logs.ps1

```diff
@@ -1,6 +1,19 @@
 $ProgressPreference = "SilentlyContinue"
 
-$lockedFiles = "kubelet.err.log", "kubelet.log", "kubeproxy.log", "kubeproxy.err.log", "azure-vnet-telemetry.log", "azure-vnet.log", "csi-proxy.log", "csi-proxy.err.log"
+$lockedFiles = @(
+  "kubelet.err.log",
+  "kubelet.log",
+  "kubeproxy.log",
+  "kubeproxy.err.log",
+  "azure-vnet-telemetry.log",
+  "azure-vnet.log",
+  "network-interfaces.json",
+  "interfaces.json",
```

Comment:
We do not need these two files "network-interfaces.json" and "interfaces.json" for AzureStack.
https://github.com/jsturtevant/aks-engine/commit/77fb7889788b6713e197812780bb514dc086b59d#diff-8cdf020a107dd40b81a5ba07a72ec72d7522c9093dfeccab008012f84b671a66